src/lib/protocols/credentials/__tests__/CredentialService.test.ts

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable no-console */
```

Comment:
```suggestion
```

This is not needed in test files anymore

Reply:
Yeah. I'll remove it.